src/microstate.js

```diff
@@ -40,4 +41,26 @@ export default class Microstate {
     let { value } = reveal(this);
     return value;
   }
+
+  [SymbolObservable]() {
```

Comment:
What do you think about putting this in its own module?

```js
import ObservableMicrostate from 'microstates/observable';

class Microstate extends ObservableMicrostate {
 ///
}
```

I don't know why, but I admire the tinyness of the Microstate class.

Reply:
Me too, but I think this is an import communication to people about microstates. If they look at the code, they can tell that hey can use it with an observable. I think it'll probably end up being the primary way of using microstates. We can always pull it out before 1.0.

What do you think?

Reply:
Ok, I can see that. The main thing that I'm worried about is folks thinking it's the _only_ way to use microstates. But I'm ok with it.